packages/components/src/InputText/InputText.tsx

```diff
@@ -144,12 +136,8 @@ function InputTextInternal(
     }
   }
 
-  function resize(textArea: HTMLTextAreaElement, wrapper: HTMLDivElement) {
-    if (rowRange.min === rowRange.max) return;
```

Comment:
ah ok, I didn't have a chance to read over the code yet. I just wanted to do a few of the problematic flows I encountered on v1 of this.

yeah I'd be hesitant to remove the resize logic since that applies to normal, non toolbar multiline text inputs too. I can't remember if it's documented or not that we offer the not-growing behavior from omitting rows but I remember seeing a few instances like that in product which may be expecting or relying on that behavior